app/styles/components/post-view.scss

```diff
@@ -0,0 +1,48 @@
+.post-view {
+  font-family: serif;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets